python/generate_dependencies.sh

```diff
@@ -15,6 +15,9 @@
 BASE_REQUIREMENTS_PATH=$1
 TARGET_REQUIREMENTS_PATH=$2
 
+# Set default Python version if not provided
+PY_VERSION=${PY_VERSION:-"3.11"}
```

Comment:
Actually, should we just get rid of this entirely? All we're doing with it is using it to create a virtual environment name, we could probably just always call it `python_requirements_gen`

Reply:
Sounds good. Done.